app/cmd/plugin_search.go

```diff
@@ -72,19 +72,70 @@ func searchPlugins(plugins *client.AvailablePluginList, keyword string) []client
 	return result
 }
 
+func matchPluginsData(plugins []client.AvailablePlugin) (result []client.CenterPlugin) {
+	result = make([]client.CenterPlugin, 0)
+	jclient := &client.UpdateCenterManager{
+		JenkinsCore: client.JenkinsCore{
+			RoundTripper: pluginSearchOption.RoundTripper,
+		},
+	}
+	getCurrentJenkinsAndClient(&(jclient.JenkinsCore))
+	site, err := jclient.GetSite()
+	if err != nil {
+		return
+	}
+	pluginJclient := &client.PluginManager{
```

Comment:
There's an old comment for this. This new PR lost all comments from #132 . This is another reason why I don't want you to create PR for the same changes.

See also https://github.com/jenkins-zh/jenkins-cli/pull/132#discussion_r323026316.